### PR TITLE
Part 7 Task 1 and 2

### PR DESCRIPTION
**Pull request by alexkulesh**

### CODE REVIEW

File: Part7Task15/Task.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    public static class Task
+    {
+        private static void Main(string[] args)
+        {
+            var locale = Console.ReadLine();
+            if (locale != null && locale.Contains("ru"))
+            {
+                Console.WriteLine("Брест");
+            }
+            else if (locale != null && locale.Contains("eng"))
```
Comment (a-miakhedka) [inline Part7Task15/Task.cs:18]: Change to 'en'

File: Part7Task15P2/Task.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    {
+        private static void Main(string[] args)
+        {
+            var locale = Console.ReadLine();
+            switch (locale)
+            {
+                case "ru":
+                    Console.WriteLine("Брест");
+                    break;
+                case "eng":
```
Comment (a-miakhedka) [inline Part7Task15P2/Task.cs:20]: Same

File: Part7Task11/Task.cs [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    //Напишите консольную программу, пользователь вводит с клавиатуры число.
+    //Программа проверяет и выводит на консоль результат ""Число четное"" либо
+    //""Число нечетное"", либо ""Введено некорректное значение"".
+    public class Task
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите число");
+            int num = Convert.ToInt32(Console.ReadLine());
+            if (num % 2 == 0 && num > 1)
```
Comment (a-miakhedka) [inline Part7Task11/Task.cs:14]: Add empty line between different operations

File: Part7Task3/Task.cs [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    // положительного вещественного числа, цифра 9.
+    // 3. Определить, равна ли первая цифра дробной части заданного
+    // положительного вещественного числа четвертой цифре.
+
+    public class Task
+    {
+        private static void Main(string[] args)
+        {
+            var num = Convert.ToDouble(Console.ReadLine());
+
```
Comment (a-miakhedka) [inline Part7Task3/Task.cs:19]: Empty line

File: Part7Task3/Task.cs [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    // 3. Определить, равна ли первая цифра дробной части заданного
+    // положительного вещественного числа четвертой цифре.
+
+    public class Task
+    {
+        private static void Main(string[] args)
+        {
+            var num = Convert.ToDouble(Console.ReadLine());
+            
+        }
```
Comment (a-miakhedka) [inline Part7Task3/Task.cs:20]: remove empty task

File: PartTask15/Program.cs [hunk 6]
```diff
@@ -0,0 +1,2 @@
+﻿// See https://aka.ms/new-console-template for more information
+Console.WriteLine("Hello, World!");
```
Comment (a-miakhedka) [inline PartTask15/Program.cs:2]: Remove it

File: Part7Task13/Task.cs [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+
+            if (a < b && b < c)
+            {
+                Console.WriteLine("Неравенство a < b < c выполняется");
+            }
+            else if (b > a && a > c)
+            {
+                Console.WriteLine("Неравенство b > a > c выполняется");
+            }
+            else if (b == a || a == c || b == c)
```
Comment (a-miakhedka) [inline Part7Task13/Task.cs:24]: Leave just else statement. It isn't necessary to check it here

File: Part7Task15/Task.cs (hunk 1 above)
Comment (alexkulesh) [reply Part7Task15/Task.cs:18]: I've changed it

File: Part7Task15P2/Task.cs (hunk 2 above)
Comment (alexkulesh) [reply Part7Task15P2/Task.cs:20]: I've changed it

File: Part7Task11/Task.cs (hunk 3 above)
Comment (alexkulesh) [reply Part7Task11/Task.cs:14]: I've added empty line

File: Part7Task3/Task.cs (hunk 5 above)
Comment (alexkulesh) [reply Part7Task3/Task.cs:20]: It will be added in another commit

File: PartTask15/Program.cs (hunk 6 above)
Comment (alexkulesh) [reply PartTask15/Program.cs:2]: Removed

File: Part7Task13/Task.cs (hunk 7 above)
Comment (alexkulesh) [reply Part7Task13/Task.cs:24]: Done

File: Part7Task15/Task.cs [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+    //Пользователь вводит с консоли строку, которая может принимать 2 значения: 'ru' либо 'en'.
+    //Если она имеет значение 'ru', то вывести на консоль слово ""Брест"", а если имеет
+    //значение 'en' – то ""Brest"".  Решите задачу через if, через switch-case (должно быть 2 решения).
+    //Обработайте ситуацию, когда введенное значение не равно 'ru' либо 'en'.
+    public static class Task
+    {
+        private static void Main(string[] args)
+        {
+            var locale = Console.ReadLine();
+            if (locale != null && locale.Contains("ru"))
```
Comment (a-miakhedka) [inline Part7Task15/Task.cs:14]: Add space between different logic

File: Part7Task15P2/Task.cs [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+    //Пользователь вводит с консоли строку, которая может принимать 2 значения: 'ru' либо 'en'.
+    //Если она имеет значение 'ru', то вывести на консоль слово ""Брест"", а если имеет
+    //значение 'en' – то ""Brest"".  Решите задачу через if, через switch-case (должно быть 2 решения).
+    //Обработайте ситуацию, когда введенное значение не равно 'ru' либо 'en'.
+    public class Task
+    {
+        private static void Main(string[] args)
+        {
+            var locale = Console.ReadLine();
+            switch (locale)
```
Comment (a-miakhedka) [inline Part7Task15P2/Task.cs:15]: same

File: Part7Task17/Program.cs [hunk 10]
```diff
@@ -0,0 +1,3 @@
+﻿// See https://aka.ms/new-console-template for more information
```
Comment (a-miakhedka) [inline Part7Task17/Program.cs:1]: Remove class

File: Part7Task4/Task.cs [hunk 11]
```diff
@@ -0,0 +1,46 @@
...
+            
+            WriteLine("числа x, y, z равны между собой - " + areEqual);
+            
+            if ((x > 0 && y < 0 && z < 0) || (x < 0 && y > 0 && z < 0) || (x < 0 && y > 0 && z < 0))
+            {
+                isPositive = true;
+            }    
+            
+            WriteLine("только одно из целых чисел x, y, z положительно - " + isPositive);
+
```
Comment (a-miakhedka) [inline Part7Task4/Task.cs:43]: Remove empty line

**Review by a-miakhedka — APPROVED** [review]

**Pull request closed without merging**